### DISCUSSION TITLE
Can I submit issues or pull requests for the Next.js official documentation website code?

### DISCUSSION THREAD

**Discussion by ShinChanU** [vercel/next.js#47275] [Help]
### Summary

Hello. While examining the Next.js official documentation, I noticed that some of the CSS values are not correct and I would like to modify them. What do I need to do if I want to make the CSS changes myself? 

Is there a separate repository for the source code of the official documentation website? Sorry if an easy answer is available.

**Comment by guitheengineer** [comment]
The docs are in this same repo (in a `/docs` folder). You basically have to find where there's something wrong, fork the repo, commit the changes and make a PR pointing to the canary branch.

See example: https://github.com/vercel/next.js/pull/45073

**Reply by ShinChanU** [reply]
First of all, thank you for your previous answer. 

However, upon checking the "docs" folder, I found only Markdown files. I am looking for the source code (HTML, CSS, JS) that makes up the official Next.js website, not just the contents (examples, usage, etc.) in the "docs" folder. 
Is it not possible to contribute to such source code?
